debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/TablesWithoutPrimaryKeyIT.java

```diff
@@ -42,6 +42,28 @@ public void before() throws SQLException {
     public void shouldProcessFromSnapshot() throws Exception {
         TestHelper.execute(STATEMENTS);
 
+        start(PostgresConnector.class, TestHelper.defaultConfig()
+                .with(PostgresConnectorConfig.SNAPSHOT_MODE, SnapshotMode.INITIAL_ONLY)
+                .with(PostgresConnectorConfig.SCHEMA_INCLUDE_LIST, "nopk")
+                .build());
+        assertConnectorIsRunning();
+
+        final int expectedRecordsCount = 1 + 1 + 1;
+
+        TestConsumer consumer = testConsumer(expectedRecordsCount, "nopk");
+        consumer.await(TestHelper.waitTimeForRecords(), TimeUnit.SECONDS);
+        final Map<String, List<SourceRecord>> recordsByTopic = recordsByTopic(expectedRecordsCount, consumer);
+        Assertions.assertThat(recordsByTopic.get("test_server.nopk.t1").get(0).keySchema().field("pk")).isNotNull();
```

Comment:
We usually import `assertThat` statically.

Reply:
I just copied code.